packages/wonder-blocks-dropdown/src/components/multi-select.tsx

```diff
@@ -478,12 +547,16 @@ const MultiSelect = (props: Props) => {
         const dropdownOpener = (
             <IDProvider id={id} scope="multi-select-opener">
                 {(uniqueOpenerId) => {
+                    const hasAriaRequired = ariaRequired || !!required;
                     return opener ? (
                         <DropdownOpener
                             id={uniqueOpenerId}
+                            error={hasError}
                             aria-controls={dropdownId}
                             aria-haspopup="listbox"
+                            aria-required={hasAriaRequired}
```

Comment:
I'm going to remove setting `aria-required` since the opener is a button and aria-required isn't allowed on buttons 😅 Found this in failing jest-axe tests while testing the snapshot in webapp (I'll add some jest-axe tests for the dropdown components too!) 

If we switch these to use role=combobox instead, [aria-required would be supported](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-required#associated_roles)! (WB-1671)

Reply:
Removed setting aria-required and added jest-axe tests! I did the same thing for SingleSelect in the [SingleSelect validation PR](https://github.com/Khan/wonder-blocks/pull/2376/commits/7f9b154c4ef875f3b47281343af4b92b04e6e56b)

I created WB-1807 to set `aria-required` once WB-1671 is done!

Reply:
@jandrade FYI! I realize now that this thread doesn't show in the PR diff since it's on code I removed 😅